### PR TITLE
update jenkinsio charts and readme

### PR DESCRIPTION
**Pull request by SarathChandra24**
This PR fixes the deployment error for chart jenkinsio which throws 
``` 
Error: INSTALLATION FAILED: 1 error occurred:
	* Ingress.extensions "jenkinsio" is invalid: [spec.rules[0].http.paths[0].backend.service.name: Required value, spec.rules[0].http.paths[1].backend.service.name: Required value]
```

Also updates the outdated instructions on https://github.com/jenkins-infra/helm-charts/tree/main/charts/jenkinsio#running-this-yourself

https://github.com/jenkins-infra/helm-charts/issues/944

### CODE REVIEW

**Review by dduportal — CHANGES_REQUESTED** [review]
Thanks for the proposal but there are many things to rework before landing this:

- You don't provide any reproduction of the error message mentioned in your PR body. Could you share how to reproduce (we usually tend to describe problem with an issue before jumping immediately to code) so us, maintainers, can either guide you on fixing your configuration or to scope the fix to code?
- The proposed change in the code is invalid and will break our usage of the chart (see line suggestion). We cannot accept it as it.
- There are no unit tests (ref. https://github.com/jenkins-infra/helm-charts/tree/main/charts/jenkinsio/tests) implementing the behavior you want: you might want to implement unit tests underlying the problem you are having, before trying to fix it so it would help maintaining this chart over time


=> The documentation changes are really useful: do you mind splitting your PR into 2 PRs:

- One for documentation (with a patch increase in the chart version)
- Another for the "ingress" problem you seem to have

File: charts/jenkinsio/templates/ingress.yaml
Comment (SarathChandra24) [reply charts/jenkinsio/templates/ingress.yaml:38]: Yes I will split this one into 2 PR's and I will close this PR.

For the serviceName change I will update the values.local.yaml in readme to have serviceName. I will create a new issue for this one.

**Comment by SarathChandra24** [comment]
I will update this PR with only documentation changes. will create a new issue with the error that I am getting with values.local.yaml

**Pull request closed without merging**
